File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/local/MeteoriteService.java
package com.antonio.samir.meteoritelandingsspots.service.local;

import androidx.lifecycle.LiveData;

import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

import java.util.List;

public interface MeteoriteService {

    LiveData<List<Meteorite>> getMeteorites();

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/local/MeteoriteService.kt
package com.antonio.samir.meteoritelandingsspots.service.local

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import com.antonio.samir.meteoritelandingsspots.model.Meteorite

interface MeteoriteService {

    fun loadMeteorites(): LiveData<List<Meteorite>>

    fun getMeteoriteById(id: String): LiveData<Meteorite>?

    fun addressStatus(): MutableLiveData<String>
}
